packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/service/service.controller.js

```diff
@@ -31,10 +31,10 @@ export default class XdslModemServiceCtrl {
       .get({
         xdslId: this.$stateParams.serviceName,
       })
-      .$promise.then(({ xdslTask }) => {
+      .$promise.then(({ data }) => {
```

Comment:
You could have 
```suggestion
      .$promise.then(({ data: xdslTask }) => {
```
to avoid having to rewrite every usage  of xdslTask

Reply:
Thanks for the advice.